src/pybind/mgr/dashboard_v2/frontend/src/app/shared/empty/empty.component.html

```diff
@@ -0,0 +1 @@
+<!-- Empty Component, delete me! -->
```

Comment:
Which purpose serves this empty component?

Reply:
Should be deleted, in the future, by the developer that implements the first route / page.

Reply:
How about adding this information to the comment then?